cfgov/jinja2/v1/_includes/organisms/email-popup/base.html

```diff
@@ -39,17 +39,17 @@ <h2>Buying a home?</h2>
                                required>
                     </div>
                     <p>
-                        <a href="/owning-a-home/privacy-act-statement/"
+                        <a href="{{ privacy_link_url }}"
                            target="_blank"
                            rel="noopener noreferrer">
-                            View Privacy Act statement
+                            {{ privacy_link_text }}
```

Comment:
Are we sure it's not a mistake that the "View Privacy Act statement" text differs across apps? Seems like they should/could be the same site-wide.

Reply:
I agree this could be the same. This configurability was based on input from stakeholders; I will double-check.

Reply:
Per feedback, we are going to standardize on "Privacy Act statement" everywhere. Updated in dbded65.